### PR TITLE
[2017-charlotte] Update welcome page and sponsor

### PR DESCRIPTION
**Pull request by hemersonvarela**

### CODE REVIEW

File: content/events/2017-charlotte/program.md [hunk 1]
```diff
@@ -1,348 +1,8 @@
 +++
-date = "2016-03-06T21:28:07-06:00"
-title = "program"
+date = "2017-01-10T10:34:20-04:00"
+title = "agenda"
```
Comment (phrawzty) [inline content/events/2017-charlotte/program.md]: It's called "program" everywhere else (including [elsewhere in this PR](https://github.com/devopsdays/devopsdays-web/pull/1658/files#diff-021f8cbcee9876e5bde59edacfa6b638R22)) - are you sure about the name change here?

Comment (mattstratton) [inline content/events/2017-charlotte/program.md]: Dang it. We just stepped on each other.

Comment (mattstratton) [inline content/events/2017-charlotte/program.md]: FWIW, it just changes the title on the page itself. The menu doesn't change.

Comment (hemersonvarela) [inline content/events/2017-charlotte/program.md]: I have updated title

**Pull request merged**